iOS/G3MiOSSDK/Commons/Rendererers/PlanetRenderer.cpp

```diff
@@ -32,24 +32,29 @@
 #include "Layer.hpp"
 #include <algorithm>
 
+#include "MaxLevelTileLODTester.hpp"
+#include "ProjectedCornersDistanceTileLODTester.hpp"
+#include "TimedTileLODTester.hpp"
+#include "MaxFrameTimeTileLODTester.hpp"
+
```

Comment:
As far as I remember, no defaults of these type are handled from the Renderers.  The "default" LODTester has to be created from the builders.   PlanetRenderer should throws an exception if no tester is provided.